tests/robottelo/test_cli.py

```diff
@@ -1,8 +1,13 @@
-import mock
+import six
 import unittest2
 
 from robottelo.cli.base import Base
 
+if six.PY2:
+    import mock
+else:
+    from unittest import mock
```

Comment:
Good point. I thought on this already, but still thinking if we should update the setup.py to install only if is python 2.